src/datachain/lib/file.py

```diff
@@ -242,6 +242,30 @@ def __init__(self, **kwargs):
         self._catalog = None
         self._caching_enabled: bool = False
 
+    def as_text_file(self) -> "TextFile":
+        """Convert the file to a `TextFile` object."""
+        if isinstance(self, TextFile):
+            return self
+        file = TextFile(**self.model_dump())
```

Comment:
Should we add check for the file content for example if it is possible to dump a file as text or not? I don't think we benefit from dumping certain binary files as text file IMO.

Reply:
Hm... For this we need to download file first. This helper is needed for user to be able to use methods from `TextFile` or `VideoFile` models, for example, to split video into frames. I suppose users know what they are doing, otherwise they'll get an error.